Makefile

```diff
@@ -46,4 +48,12 @@ release:
 	opam publish submit $(NAME_VERSION)
 	rm -rf $(NAME_VERSION)
 
+install:
+	ocamlfind remove ppx_deriving
+	grep -E '^[[:space:]]+' ppx_deriving.install | \
```

Comment:
On 10/04/2015 12:22, Vincent wrote:
> I've looked after ocp-build, I have big hopes for it,

Abandon all hope, ye who enter here…

but right now
> there is a clear lack of documentation for the tool which makes it a
> no-go for an use in a professional context.

Yes, obviously. That's why I said "try to", just in case it would work
here, you would be done with it :)

-- 
Vincent